### DISCUSSION TITLE
Can current demand and current accumulative cost be reloaded?

### DISCUSSION THREAD

**Discussion by deltamelter** [BottlecapDave/HomeAssistant-OctopusEnergy#591] [Q&A]
My HA server was one of the things powered down during the Saving Session today, the home mini was left connected as was the internet.  After the event everything came back except for the current demand and current cumulative cost during this period.
I tried using the service to refresh consumption but that is not actually missing and gave index out of range error.
Is it possible to recalculate these?
![image](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/27061628/a2470058-89c8-4a39-a29c-2634e87a45d4)

**Comment by BottlecapDave** [comment]
Hello. The current demand is basically a "live" view so not supported. The current accumulative consumption sensor value will update to show the missed data (as long as it updates with the same day). The HA sensors don't support loading historic data (see previous consumption sensor) so this would not be possible I'm afraid.
